coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,9 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter_name, available_filters):
+        self.filter_name = filter_name
+        self.available_filters = available_filters
+        message = {self.filter_name +
+                   ' is an invalid filter. Available filters: ' +
+                   ' ,'.join(self.available_filters)}
```

Comment:
@aashutoshrathi Because you `join` them here again ;) So will get all single characters separated by commas, since you are already passing the joined string to this constructor. Also `self.available_filters` should still be a sequence of filters and no joined string. So only `join` here

And change `' ,'` to `', '`